tests/test_browser.py

```diff
@@ -73,9 +74,9 @@ def do_GET(s):
 
 
 def shell_with_script(shell_file, output_file, replacement):
-  with open(path_from_root('src', shell_file)) as input:
-    with open(output_file, 'w') as output:
-      output.write(input.read().replace('{{{ SCRIPT }}}', replacement))
+  shell = read_file(path_from_root('src', shell_file))
+  with open(output_file, 'w') as output:
+    output.write(shell.replace('{{{ SCRIPT }}}', replacement))
```

Comment:
We have `read_file`... Would it be good to have `write_file` too?

Reply:
Hmm .. we do have `create_file` already.. I think I can use that here.. thanks!